server/api/auto_respond.go

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) 2019-present Mattermost, Inc. All Rights Reserved.
+// See License for license information.
+
+package api
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/mattermost/mattermost-server/v5/model"
+
+	"github.com/mattermost/mattermost-plugin-mscalendar/server/mscalendar"
+)
+
+func (api *api) autoRespond(w http.ResponseWriter, req *http.Request) {
```

Comment:
I think the method name should be more descriptive like this. Then the API route can be `set-autorespond-message`.

```suggestion
func (api *api) setAutorespondMessage(w http.ResponseWriter, req *http.Request) {
```

Reply:
To keep in convention with other naming ("auto" and "respond" being separate words), the API route would be `set-auto-respond-message`, the method name would be `setAutoRespondMessage`, and the constant in `config/const.go` would be `PathSetAutoRespondMessage`, but those are getting lengthy.